test/vcf/record_test.cpp

```diff
@@ -35,54 +35,54 @@ namespace ebi
                 {},
                 { "Sample1" }}};
             
-        source->meta_entries.emplace("FORMAT",
+        source->meta_entries.emplace(vcf::FORMAT,
             vcf::MetaEntry{
                 1,
-                "FORMAT",
+                vcf::FORMAT,
                 {
-                    { "ID", "GT" },
-                    { "Number", "1" },
-                    { "Type", "String" },
-                    { "Description", "Genotype" }
+                    { vcf::ID, vcf::GT },
+                    { vcf::NUMBER, "1" },
+                    { vcf::TYPE, vcf::STRING },
+                    { vcf::DESCRIPTION, "Genotype" }
                 },
                 source
         });
            
-        source->meta_entries.emplace("FORMAT",
+        source->meta_entries.emplace(vcf::FORMAT,
             vcf::MetaEntry{
                 1,
-                "FORMAT",
+                vcf::FORMAT,
                 {
-                    { "ID", "DP" },
-                    { "Number", "1" },
-                    { "Type", "Integer" },
-                    { "Description", "Read depth" }
+                    { vcf::ID, vcf::DP },
+                    { vcf::NUMBER, "1" },
+                    { vcf::TYPE, vcf::INTEGER },
+                    { vcf::DESCRIPTION, "Read depth" }
                 },
                 source
         });
 
-        source->meta_entries.emplace("INFO",
+        source->meta_entries.emplace(vcf::INFO,
             vcf::MetaEntry{
                 1,
-                "INFO",
+                vcf::INFO,
                 {
-                    { "ID", "AN" },
-                    { "Number", "1" },
-                    { "Type", "Integer" },
-                    { "Description", "Allele number" }
+                    { vcf::ID, vcf::AN },
+                    { vcf::NUMBER, "1" },
+                    { vcf::TYPE, vcf::INTEGER },
+                    { vcf::DESCRIPTION, "Allele number" }
                 },
                 source
         });
            
-        source->meta_entries.emplace("INFO",
+        source->meta_entries.emplace(vcf::INFO,
             vcf::MetaEntry{
                 1,
-                "INFO",
+                vcf::INFO,
                 {
-                    { "ID", "AF" },
-                    { "Number", "A" },
-                    { "Type", "Float" },
-                    { "Description", "Allele frequency" }
+                    { vcf::ID, vcf::AF },
+                    { vcf::NUMBER, vcf::A },
```

Comment:
except for this usage, which is ok, revert all the other usages of "A" in this file as well